.babelrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  presets: [
+    [
+      "@babel/preset-env",
+      {
+        exclude: ["transform-async-to-generator", "transform-regenerator"]
+      }
+    ],
+    "@babel/preset-react",
+    "@babel/preset-flow"
+  ],
+  plugins: [
+    "@babel/plugin-proposal-class-properties",
+    "babel-plugin-async-to-promises"
+  ],
+  env: {
+    test: {
```

Comment:
IDK why "babel-plugin-async-to-promises" does not work in test files. If we don't use "transform-async-to-generator" for test files, using `async await` will break the test 🤔